add design

Aman-8955 opened the pull request:
webpage:- 
Added some design like fix image size and added come colors and design and make it fully responsive for all devices.
demo is attached.
![demo](https://github.com/user-attachments/assets/1dca6431-0f94-4cb1-ba3c-2b3f78b834a7)
issue related;- #21

surhud004 commented on views/foodish.ejs:
```diff
@@ -1,58 +1,75 @@
-        <div class="row align-items-center">
-            <div class="col">
+    <div>
+        <div id="nav">
```
Can we add more vertical padding to the navbar?

surhud004 reviewed (commented):
Overall, the changes look good to me. The responsive design is amazing!

Can we add some more colors based on a palette? Feel free to choose any modern, minimalist, food/restaurant related color palette.
Here's an example with `darkslategray` and `burlywood` -

![example](https://github.com/user-attachments/assets/2ff7853a-6cdf-418d-817e-ee806f8388fd)

Aman-8955 commented:
I changed the navigation alignment and added padding. I also added a functionality for color changing. I created two variables—if you want to change the color, just modify those, and the whole webpage color will automatically update.
![screencapture-localhost-3000-2024-09-25-12_05_26](https://github.com/user-attachments/assets/8ea9ba02-623f-4a7b-9ee0-b2746e0c890c)
